styletesla.css

```diff
@@ -1,97 +1,772 @@
 .body {
-    max-width: 1440px;
-    margin: 0 auto;
-    }
+  max-width: 1440px;
+  margin: 0 auto;
+}
+
 .mainpage {
-	height: 100vh;
-        }
+  height: 100vh;
+}
+
 .header {
-	height: 100px;
-	display: flex;
-	justify-content: space-between;
-	align-items: center;
+  height: 10vh;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  padding-left: 47px;
 }
+
 .navigation a {
-	display: inline;
-    margin-bottom: 67px;
-    cursor: pointer;
-    margin-right: 44px;
+  font: 15px/18px montserratsemibold, sans-serif;
+  cursor: pointer;
+  margin-right: 44px;
 }
+
 .navigation__link_active {
-	border-bottom: 1px solid red;
-	margin-right: 63px;
-	color: red;}
+  padding-bottom: 5px;
+  border-bottom: 1px solid #e51937;
+  color: #e51937;
+}
 
 .menu a {
-	margin-right: 30px;
+  cursor: pointer;
+  font: 15px/18px montserratsemibold, sans-serif;
+  margin-right: 30px;
 }
+
 .main {
-	background: url('img/tesla-wordmark.png') top no-repeat;
-	display: flex;
-	align-items: center;
-	flex-direction: column;
+  background: url('img/tesla-wordmark.png') top no-repeat;
+  background-size: contain;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
 }
 
-.tesla{ 
-	/*width: 67%;*/
-	max-width: 968px;
+.main__title {
+  font: 9vh/11vh montserrat, sans-serif;
+  font-weight: 600;
+  margin: 0;
+}
+ 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla-12.04/pullRequest?prid=5512124)